Framework/API/src/MatrixWorkspace.cpp

```diff
@@ -676,6 +673,14 @@ void MatrixWorkspace::getXMinMax(double &xmin, double &xmax) const {
         xmax = xback;
     }
   }
+  if (m_indexInfo->size() != m_indexInfo->globalSize()) {
```

Comment:
Very minor point. If this is the kind of query that would be used repeatedly, would it be better to add something like `bool isDistributed() const` to `IndexInfo`?

Reply:
For now just here. I do not see this being used a lot.